datafusion/optimizer/src/simplify_expressions/expr_simplifier.rs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the Apache Software Foundation (ASF) under one
+// Licensed to the Apac
```

Comment:
this seems an unintended change

```suggestion
// Licensed to the Apache Software Foundation (ASF) under one
```